### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by observingClouds**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +27,9 @@ Klöwer, M., Razinger, M., Dominguez, J. J., Düben, P. D., & Palmer, T. N. (202
 
 ## How to install
 
-`pip install git+https://github.com/observingClouds/xbitinfo.git`
+`conda install -c conda-forge xbitinfo`
 
-`pip install git+https://github.com/observingClouds/xbitinfo.git#egg=xbitinfo[complete]`
+`pip install xbitinfo`
```
Comment (aaronspring) [inline README.md:32]: pip install xbitinfo[complete] installs all optional dependencies but not needed here

**Review by aaronspring — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (observingClouds) [reply README.md:32]: @aaronspring , have you tried if that works?

Comment (aaronspring) [reply README.md:32]: I get a netcdf error on levante which seems unrelated:
```>>> import xbitinfo
/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/xarray/backends/cfgrib_.py:27: UserWarning: Failed to load cfgrib - most likely there is a problem accessing the ecCodes library. Try `import cfgrib` to get the full error message
  warnings.warn(
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/xbitinfo/__init__.py", line 4, in <module>
    from .bitround import jl_bitround, xr_bitround
  File "/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/xbitinfo/bitround.py", line 1, in <module>
    import xarray as xr
  File "/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/xarray/__init__.py", line 1, in <module>
    from . import testing, tutorial, ufuncs
  File "/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/xarray/tutorial.py", line 13, in <module>
    from .backends.api import open_dataset as _open_dataset
  File "/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/xarray/backends/__init__.py", line 9, in <module>
    from .h5netcdf_ import H5NetCDFStore
  File "/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/xarray/backends/h5netcdf_.py", line 25, in <module>
    from .netCDF4_ import (
  File "/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/xarray/backends/netCDF4_.py", line 33, in <module>
    import netCDF4
  File "/work/mh0727/m300524/conda-envs/ML/lib/python3.9/site-packages/netCDF4/__init__.py", line 3, in <module>
    from ._netCDF4 import *
ImportError: libmpi.so.12: cannot open shared object file: No such file or directory

Comment (aaronspring) [reply README.md:32]: and no `pip install xbitinfo[complete]` doesnt work :(

Comment (aaronspring) [reply README.md:32]: `mamba install xbitinfo` also fails

```Encountered problems while solving:
  - package xbitinfo-0.0.2-pyhd8ed1ab_0 requires pyjulia >=0.5.7, but none of the providers can be installed

Comment (aaronspring) [reply README.md:32]: but this PR is still what we need

Comment (observingClouds) [reply README.md:32]: Strange, so `mamba create -n xbitinfo-test python=3.9 xbitinfo` works on my Mac

**Pull request merged**